### DISCUSSION TITLE
Home Assistant MQTT Discovery

### DISCUSSION THREAD

**Discussion by dcgrove** [ESPresense/ESPresense#10] [Ideas]
Any plans to support discovery in home assistant for devices? This would greatly simplify the setup in homeassistant as they could automatically be added as device trackers.

https://www.home-assistant.io/docs/mqtt/discovery/

**Comment by DTTerastar** [comment]
YES, I almost did it this weekend.

**Reply by dcgrove** [reply]
Awesome. I look forward to it!
